docs/kits.rst

```diff
@@ -73,13 +128,17 @@ command palette. The following steps are taken to find available kits:
         CMake Tools will not delete entries from ``cmake-kits.json``, only add
         and update existing ones.
 
+.. _kit-types:
+
 Kit Types
 =========
 
 CMake defines three types of kits: *compiler kits*, *Visual Studio kits*, and
 *toolchain file kits*. They are distinguished by the properties present in
```

Comment:
I think that `Compiler` and `Toolchain` should be written in upper case.

Reply:
In the phrase `Toolchain Kit` and `Compiler Kit`, probably.

Reply:
yeah